.github/workflows/nodejs.yaml

```diff
@@ -0,0 +1,50 @@
+name: Installing dependencies
+'on':
+  workflow_dispatch: null
+  push:
+    branches:
+      - main
+      - develop
+    tags:
+      - v*
+jobs:
+  nodejs:
+    name: Tests of releases NodeJS
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 10.x
+          - 12.x
+          - 14.x
+          - 15.x
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Install nodejs dependencies ${{ matrix.node-version }}'''
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ matrix.node-version }}'
+      - run: |
+          npm install -D
+          npm install
+  build_app:
+    name: Build App (desktop)
+    needs:
+      - nodejs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - run: npm install
```

Comment:
Is the `npm install` duplicated between the job and the build_app workflow? Again, we need to cache dependencies here. Also, we are migrating to Yarn, so we should use yarn instead of NPM.